### PR TITLE
Add MongoDB server

### PR DESCRIPTION
**Pull request by KamillaKhabibrakhmanova**

### CODE REVIEW

File: database.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var mongoose = require('mongoose'),
```
Comment (aaghevli) [inline database.js]: Can we place this file in libs dir?

File: models/event.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+var mongoose = require('mongoose'),
+    Schema = mongoose.Schema;
+
+var EventSchema = new mongoose.Schema({
```
Comment (aaghevli) [inline models/event.js]: What do you think about using the OSDI event as the schema for persistence?

https://opensupporter.github.io/osdi-docs/events.html

File: database.js (hunk 1 above)
Comment (aaghevli) [inline database.js]: Can we place this in the libs folder?

File: models/event.js (hunk 2 above)
Comment (aaghevli) [inline models/event.js]: What do you think about using the OSDI event as the schema for persistence?

https://opensupporter.github.io/osdi-docs/events.html

If so, can we place in models/osdi/event.js ?

**Comment by KamillaKhabibrakhmanova** [comment]
@aaghevli fixed :)

File: config.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    mongoUri: process.env.MONGO_URI
```
Comment (watzkej) [inline config.js]: Can you set this to MONGODB_URI?  That's the variable that Heroku injects automatically.  I could change it on the Heroku side but I'm not sure if there are any side-effects if we move away from the default that Heroku automatically generates.

File: models/event.js (hunk 2 above)
Comment (watzkej) [inline models/event.js]: Unless this changed since you added this comment, this is the OSDI event schema.

File: server.js [hunk 4]
```diff
@@ -2,11 +2,13 @@
...
 'use strict';
 
-const Hapi = require('hapi');
+const Hapi = require('hapi'),
+Db = require('./lib/database');
 
 // Create a server with a host and port
 const server = new Hapi.Server();
 server.connection({
+    host: 'localhost',
```
Comment (watzkej) [inline server.js]: Yeah, I took this out when I was being a bit touchy with trying to get it to work on Heroku.  I somehow got the idea that this was causing the port binding to only work for localhost.  I imagine I broke local node stuff so it is good you put it back.

**Pull request merged**